Utils/Dataflow/019_esFormat/README

```diff
@@ -40,3 +40,15 @@ For input it is OK, but for output it would be better to take <index info> and
   be critical for now, but something tells me that it will.
 This task requires accurate EOMessage and EOProcess markers handling (see the
   `kafka-dataflow-markers` branch).
+
+Samples
+-------
+The 'output' directory contains two pairs of samples:
```

Comment:
'tasks' and 'datasets' are quite self-describing filenames, yet maybe add the description of what they are?

Reply:
Done.